docs/Modders/uniques.md

```diff
@@ -128,13 +128,13 @@ Simple unique parameters are explained by mouseover. Complex parameters are expl
 
 	Applicable to: Global, FollowerBelief
 
-??? example  "[relativeAmount]% [stat] from every [buildingFilter/tileFilter]"
-	Example: "[+20]% [Culture] from every [Culture]"
```

Comment:
Just out of curiosity, how did it happen to generate [Culture] twice before? Was it taken from tileFilter?

Reply:
No the buildingFilter example is "Culture"

Reply:
Thanks. 🙇🏻‍♂️